app/src/main/java/com/github/sdp_begreen/begreen/activities/MainActivity.kt

```diff
@@ -153,7 +153,19 @@ class MainActivity : AppCompatActivity() {
             }
             R.id.bottomMenuAdvice -> {
                 item.setIcon(R.drawable.ic_baseline_lightbulb)
-                replaceFragInMainContainer(AdviceFragment())
+                val adviceList= arrayListOf(
+                "Ecology helps us to understand how our actions affect the environment. It shows the individuals the extent of damage we cause to the environment.",
+                "With the knowledge of ecology, we are able to know which resources are necessary for the survival of different organisms. Lack of ecological knowledge has led to scarcity and deprivation of these resources, leading to competition.",
+                "All organisms require energy for their growth and development. Lack of ecological understanding leads to the over-exploitation of energy resources such as light, nutrition and radiation, leading to its depletion.",
+                "Ecology encourages harmonious living within the species and the adoption of a lifestyle that protects the ecology of life.",
+                "It focuses on the relationship between humans and the environment. It emphasizes the impact human beings have on the environment and gives knowledge on how we can improve ourselves for the betterment of humans and the environment.",
+                "It deals with the study of how organisms alter the environment for the benefit of themselves and other living beings. For eg, termites create a 6 feet tall mound and at the same time feed and protect their entire population.",
+                "Ecology plays a significant role in forming new species and modifying the existing ones. Natural selection is one of the many factors that influences evolutionary change.",
+                "Ecology was first devised by Ernst Haeckel, a German Zoologist. However, ecology has its origins in other sciences such as geology, biology, and evolution among others.",
+                "Habitat ecology is the type of natural environment in which a particular species of an organism live, characterized by both physical and biological features.",
+                "An organism free from the interference of other species and can use a full range of biotic and abiotic resources in which it can survive and reproduce is known as its fundamental niche."
+                )
+                replaceFragInMainContainer(AdviceFragment.newInstance(adviceList))
```

Comment:
Those advices are very nice! 
I'm currently working on adding and retrieving data from firebase, if you want I can store those advices in the database and define a new function in `FirebaseDB` that retrieves the advices stored on the database: that way, we could update those advices anytime in the database and the users would see the update when they open this activity.
Tell me what you think about that :)

Reply:
Thanks for your comments and review Luca! Your suggestion definitely makes sense. I suggest to plan this refactoring task for the next sprint. What do you think?

Reply:
I just finished my second task and I have a bit of time left until friday so I can definitely add myself a new task to do that once your changes will be pushed to master!

Reply:
That's nice! Thanks for your hard work and devotion. I am obviously at your disposal may you need any help or detailed information regarding the code I wrote.